### DISCUSSION TITLE
Live grep fuzzy by default

### DISCUSSION THREAD

**Discussion by axkirillov** [ibhagwan/fzf-lua#1299] [Q&A] [answered]
I didn't find an option which would enable fuzzy search in live grep by default. Right now it always starts in regex search. Is there such an option? Or should I add a PR for it?

**Comment by abkrm** [comment, marked as answer]
As far as I know there isn't, because that's the expected behavior of live_grep. But alternatively you could use `grep` or `grep_project`, I personally use `grep_project` because of that very reason.

**Reply by axkirillov** [reply]
I see, thank you! I think that solves my problem 😄
